### ISSUE TITLE
Question about training set and validation set

### ISSUE THREAD

**Issue by jwde-code** [WarranWeng/ET-Net#2]
This is really a job. However i have a question about whether you have divided a training set and a validation set for training ? It seem that you did not release training code.

**Comment by WarranWeng** [comment]
Yes, we did. We followed the instructions from this paper 'Reducing the Sim-to-Real Gap for Event Cameras' to make our training dataset. We use the provided HQF dataset for validatng. Please check the paper for details.
Thanks for your attention.

**Comment by jwde-code** [comment]
Thanks for your reply. 🙂

I specifically mean that whether you have divided  the 280 sequences, which are maked from the paper 'Reducing the Sim-to-Real Gap for Event Cameras',  to a training set and a validation set when you were training.
Or you just take the whole 280 sequences for training and another dataset for validation (not testing)?  And what about the rules of division for training set and a validation set?

The additional question is that the testing set of office_zigzag in IJRR is just about 11 seconds long, whose release I find in official site, but i see that you test from 5s to 12s in supplementary materials.

Looking forward to your reply again.

**Comment by WarranWeng** [comment]
Sorry for the late reply. I am busy with other things recently.
1. Actually, we choose HQF dataset for validation. We donot divide the 280 sequences, which serve as a whole training set as in E2VID+(ECCV2020).
2. 5s to 12s means 5s to the end. Sorry for this issue.
